components/ExperimentsTable.tsx

```diff
@@ -0,0 +1,115 @@
+import { format } from 'date-fns'
+import debugFactory from 'debug'
+import { toInt } from 'qc-to_int'
+import React, { MouseEvent, useState } from 'react'
+
+import Menu from 'semantic-ui-react/dist/commonjs/collections/Menu'
+import Table from 'semantic-ui-react/dist/commonjs/collections/Table'
+
+import { Experiment } from '@/models/index'
+
+const debug = debugFactory('abacus:components/ExperimentsTable.tsx')
+
+const PER_PAGE_DEFAULT = 25
+
+interface Props {
+  experiments: Experiment[]
+  // The zero-based page of data to display. Defaults to 0.
+  initialPage?: number
+  // The number of rows/items to display per page. Defaults to PER_PAGE_DEFAULT.
+  perPage?: number
+}
+
+const ExperimentsTable = (props: Props) => {
+  debug('ExperimentsTable#render')
+  const { experiments, initialPage = 0, perPage = PER_PAGE_DEFAULT } = props
+  const [page, setPage] = useState(initialPage)
+
+  const pageCount = Math.ceil(experiments.length / perPage)
+  const pageNums = Array.from(Array(pageCount).keys()).map((num) => num + 1)
+
+  function handlePageStep(event: MouseEvent<HTMLAnchorElement>) {
+    debug('ExperimentsTable#handlePageStep')
+    event.preventDefault()
+    event.stopPropagation()
+    setPage(page + toInt(event.currentTarget.dataset.step))
+  }
+
+  function handleSetPage(event: MouseEvent<HTMLAnchorElement>) {
+    debug('ExperimentsTable#handleSetPage')
+    event.preventDefault()
+    event.stopPropagation()
+    setPage(toInt(event.currentTarget.dataset.page))
+  }
+
+  const beginIdx = perPage * page
+  const endIdx = beginIdx + perPage
+  const experimentsToRender = experiments.slice(beginIdx, endIdx)
+  return (
+    <Table>
+      <Table.Header>
+        <Table.Row>
+          <Table.HeaderCell>ID</Table.HeaderCell>
+          <Table.HeaderCell>Name</Table.HeaderCell>
+          <Table.HeaderCell>Started</Table.HeaderCell>
+          <Table.HeaderCell>Ends</Table.HeaderCell>
```

Comment:
These can be in the past or future. Let's stick to start/end.

```suggestion
          <Table.HeaderCell>Start</Table.HeaderCell>
          <Table.HeaderCell>End</Table.HeaderCell>
```